src/api/nft-image/nft-image.service.ts

```diff
@@ -0,0 +1,46 @@
+import { PrismaClient } from "@prisma/client";
+import { NftImage } from "./nft-image.types";
+import { Nft } from "../nft/nft.types";
+
+const prisma = new PrismaClient()
+export const getAllNftImage = async () => {
+  const nftImage = await prisma.nftImage.findMany({
+    include:{
+      imageForNft:{
+        select: {
+          nftImage: {
+            select: {
+              url: true
+            }
+          }
+        }
+      }
+    }
+  }) 
+  return nftImage
+}
+export const getSigleNftImage = async (id:string) => {
+  const nftImage = await prisma.nftImage.findUnique({
+    where:{
+      id
+    }
+  }) 
+  return nftImage
+}
+
+export const createNftImage = async (data: NftImage ) => {
+  const nftImage = await prisma.nftImage.create({
+    data: {...data}
```

Comment:
I consider the param should be named inputData for avoiding misunderstandings with data inserted to DB.